.github/workflows/pull_request.yml

```diff
@@ -22,4 +22,3 @@ jobs:
   markdown_link_check:
     name: Markdown Link Check
     uses: StanfordBDHG/.github/.github/workflows/markdown-link-check.yml@v2
-  
```

Comment:
This diff is not necessary.

Reply:
I think this came directly from the template I used for the repo generation, but I'm not sure. Will fix!